test-integration-requirements.txt

```diff
@@ -0,0 +1 @@
+pytest-operator
```

Comment:
We probably want to put a basic version requirement on this

Reply:
Any preference on what to pin to?  I don't see it pinned in our others.  `<1.0`?  

With it being something we control I'm fine with leaving it unpinned but don't feel strongly either way.

Reply:
Even if it's under our control, we'll want to pin a version just so that we can make breaking changes without breaking the world sometime down the line. I would probably pin it to `<0.9`.

This also reminds me that I should update the other PR I've got open to include versions on the `requirements.txt`'s :stuck_out_tongue: